### ISSUE TITLE
[Feature request] Add tags to histories and/or invocations when invoking workflow

### ISSUE THREAD

**Issue by simonbray** [galaxyproject/galaxy#17562]
Not sure how used this feature is, so perhaps this is niche, but I'm currently launching workflows with multiple datasets selected under an input in order to create multiple workflow invocations at once, which is very handy.

To keep things tidy, I'm using the `Send results to new history` option to run each invocation in a separate history. It would be nice to be able to add a tag to all the created histories or invocations, to be able to easily track which belong together.

(Comparable to #8710 / #13885 I guess, but for workflows rather than tools.)

**Comment by mvdbeek** [comment]
This is a duplicate of https://github.com/galaxyproject/galaxy/issues/17103, which I think we shouldn't do

**Comment by simonbray** [comment]
It is not the same as #17103, which suggests tagging output datasets of a single invocation. I would like to be able to tag the multiple histories and invocations created by a single "launch" (not invocation, but not sure what word is appropriate) of a workflow in the UI, so they are linked together. If they can be linked in another way that doesn't involve tags that would be great as well.

**Comment by simonbray** [comment]
> It is not the same as #17103, which suggests tagging output datasets of a single invocation. I would like to be able to tag the multiple histories and invocations created by a single "launch" (not invocation, but not sure what word is appropriate) of a workflow in the UI, so they are linked together. If they can be linked in another way that doesn't involve tags that would be great as well.

Can this be reopened please?

**Comment by simonbray** [comment]
I agree with John's comment, and being able to filter histories by invocation id will be useful (thanks for implementing it!)
but I don't think it's related to what I am describing here.
